src/libraries/System.Text.Json/tests/Serialization/PropertyVisibilityTests.cs

```diff
@@ -2161,20 +2165,193 @@ private struct SmallStructWithValueAndReferenceTypes
         {
             public string MyString { get; }
             public int MyInt { get; set; }
+            public bool? MyBool { get; set; }
             public Point_2D_Struct MyPointStruct { get; set; }
 
             [JsonConstructor]
             public SmallStructWithValueAndReferenceTypes(
                 Point_2D_Struct myPointStruct,
                 string myString = "Default",
-                int myInt = -1)
+                int myInt = -1,
+                bool? myBool = true)
             {
                 MyString = myString;
                 MyInt = myInt;
+                MyBool = myBool;
                 MyPointStruct = myPointStruct;
             }
         }
 
-        private class PointClass { }
+        public class PointClass { }
+
+        [Fact]
+        public static void Ignore_WhenWriting_Default_Globally()
+        {
+            var options = new JsonSerializerOptions
+            {
+                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
```

Comment:
Test is called `WhenWriting_Default` but `JsonIgnoreCondition.WhenWritingNull` is being used?